cli2/lib/FileChanges.js

```diff
@@ -0,0 +1,268 @@
+"use strict";
```

Comment:
I don't think we should check in the generated JS files. Publishing will be done after the build so they should be there in the NPM package even if we don't check these in.